app/services/lay_cards.rb

```diff
@@ -0,0 +1,12 @@
+class LayCards < Turn
+  ACTION = 'lay_cards'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.